earth_enterprise/src/common/filebundle/cachedreadaccessor_unittest.cpp

```diff
@@ -238,6 +241,8 @@ class CachedReadAccessorUnitTest : public UnitTest<CachedReadAccessorUnitTest> {
     // same block cached read
     offset *= 2;
     request_count -= offset;
+    buffer.resize(request_count);
```

Comment:
resize would truncate if reducing and push back if adding more space keeping the start of buffer at same address - the reserve would guarantee that a continuous block of memory is reserved hence no overwrite by other parts of program.